python/hail/methods/impex.py

```diff
@@ -859,11 +863,11 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
 
         Use ``delimiter='\\s+'`` to specify whitespace delimited files.
 
-    `comment` optionally causes Hail to skip any line that starts with the given
-    string. Passing ``comment='#'`` will skip any line beginning in a pound
-    sign, for example.
+    The `comment` parameter optionally causes Hail to skip any line that starts
```

Comment:
I don't like optionally used in this way. How about:

If set, the `comment` parameter causes Hail to skip lines that start ...